src/components/Tooltip/index.tsx

```diff
@@ -453,6 +475,18 @@ export function Tooltip(props: TooltipProps): ReactElement {
         return () => document.removeEventListener('scroll', delayedHideTooltip, true);
     }, [delayedHideTooltip, hideOnDocumentScroll]);
 
+    // Hide tooltip on escape
+    const escapeTooltip = (rawEvent: Event) => {
```

Comment:
Can this use the new `useWindowListener` too?

Reply:
good call!